src/api/daemon_info_test.go

```diff
@@ -0,0 +1,15 @@
+package api
+
+import (
+	"testing"
+)
+
+func TestGetDaemon(t *testing.T) {
+	daemonInfo, err := GetDaemon("dbus", "systemctl")
+
+	if err != nil {
+		t.Fatalf("Errors occured during obtaining daemon info: %s", err)
+	}
+
+	t.Logf("Name: %s, IsActive: %s, IsEnabled: %s", daemonInfo.name, daemonInfo.isActive, daemonInfo.isEnabled)
```

Comment:
Check whether deamonInfo attributes are not empty

Reply:
ok